src/controllers/expensesController.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const expensesService = require('../services/expensesService');
+const usersService = require('../services/usersService');
+
+const timestampRegex = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z$/;
+
+const get = async(req, res) => {
+  res.status(200).json(expensesService.getAll(req.query));
+};
+
+const post = async(req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  if (typeof userId !== 'number'
+      || typeof spentAt !== 'string'
+      || typeof title !== 'string'
+      || typeof amount !== 'number'
+      || typeof category !== 'string'
+      || typeof note !== 'string'
+      || !usersService.getById(userId)
+      || !timestampRegex.test(spentAt)) {
```

Comment:
Create separate function for such check
Inside `if` it looks very difficult for reading